azure-pipelines-release.yml

```diff
@@ -6,8 +6,24 @@ pool:
   demands:
     - ImageOverride -equals MMS2022TLS
 
+variables:
+  PackageSuffix: ''
+
 steps:
 
+- task: PowerShell@2
```

Comment:
You don't need to use powershell to set this variables. It can be done in the `variables` section directly via conditionals.

Reply:
good idea, done!